WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1453,6 +1454,18 @@ DialogButton                = @FontSize-big@; color: @AltC@; height: auto; paddi
 
 AttachDialogContainer       = background-color: @lighten(SecondaryBrandingColor, 94)@;
 
+ChooseFolderDialog                        = width:350px; height:422px; @borderRadius@:3px;
+ChooseFolderDialogContainer               = background-color:@lighten(SecondaryBrandingColor, 94)@;
+ChooseFolderDialogBody                    = width:320px; @Text@
+ChooseFolderDialogInputContainer          = padding-bottom:3px; padding-top:3px;
+ChooseFolderDialogInputField              = @borderRadius@:2px; @PageBg@; @SeparatorBorder@; width:320px; height:32px; margin-bottom:16px;
+ChooseFolderDialogFolderContainer         = background-color:@AppC@; overflow:auto;
+ChooseFolderDialogFolderContainerContent  = @SeparatorBorder@; padding:18px 0px 0px 16px; border-color:@SeparatorColor@; @borderRadius@:2px;
+ChooseFolderDialogFolderItem              = padding-right:0px; height:20px;
+ChooseFolderDialogFolderSelected          = background-color:@WidgetBgColor@; height:32px;
```

Comment:
Okay. But my point is why only we need to set height for selected state only and not in normal state? 

Let me know if I am missing something.